### PR TITLE
Visual improvements

### PR DESCRIPTION
**Pull request by gpantelis**
Broken long lines of code for improved readability

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against master? Unless there is a good reason otherwise, we prefer pull requests against master and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS that we support](https://www.elastic.co/support/matrix#show_os)?

### CODE REVIEW

File: core/src/main/java/org/elasticsearch/node/Node.java [hunk 1]
```diff
@@ -399,15 +412,20 @@ protected Node(final Environment environment, Collection<Class<? extends Plugin>
                 pluginsService.filterPlugins(Plugin.class).stream()
                 .map(Plugin::getCustomMetaDataUpgrader)
                 .collect(Collectors.toList());
-            final RestController restController = actionModule.getRestController();
```
Comment (jasontedor) [inline core/src/main/java/org/elasticsearch/node/Node.java]: If the line fits within a 100 columns, there is no need to reformat it.

File: core/src/main/java/org/elasticsearch/node/Node.java [hunk 2]
```diff
@@ -399,15 +412,20 @@ protected Node(final Environment environment, Collection<Class<? extends Plugin>
...
-                    threadPool, bigArrays, circuitBreakerService, namedWriteableRegistry, xContentRegistry, networkService, restController);
-            final MetaDataUpgrader metaDataUpgrader = new MetaDataUpgrader(customMetaDataUpgraders);
-            final Transport transport = networkModule.getTransportSupplier().get();
-            final TransportService transportService = newTransportService(settings, transport, threadPool,
-                networkModule.getTransportInterceptor(), localNodeFactory, settingsModule.getClusterSettings());
+            
+            final RestController restController = 
+                actionModule.getRestController();
+            final NetworkModule networkModule =
+                new NetworkModule(settings, false, pluginsService.filterPlugins(NetworkPlugin.class), threadPool, bigArrays, circuitBreakerService, namedWriteableRegistry, xContentRegistry, networkService, restController);
```
Comment (jasontedor) [inline core/src/main/java/org/elasticsearch/node/Node.java]: If we are going to reformat, it should be within the 100-column limit.

Comment (gpantelis) [inline core/src/main/java/org/elasticsearch/node/Node.java]: Very well, i'm on it

**Comment by jasontedor** [comment]
Thank you for the PR @gpantelis; we are working on reformatting the entire codebase so I do not think we need to pull this in at this time.

**Comment by jasontedor** [comment]
If you would like to pick up an issue with the adoptme label, we would welcome it.

**Comment by gpantelis** [comment]
thank you

**Comment by gpantelis** [comment]
We are working on an assigment in the university about code quality and we have selected the elasticsearch project to work on and contribute. Do you have an idea about a specific issue that we can contibute to, regarding code quality.
Thank you

**Comment by jasontedor** [comment]
Would you please send me an email? You can find my email address in my profile.

**Pull request closed without merging**